### ISSUE TITLE
Lesson cards grow if words are too long

### ISSUE THREAD

**Issue by pedro16v** [catalpainternational/canoe#829]
Surely a problem for a deployment in German 😁 

Easily solvable but we need to pick how it should break @andrewwapling @Juliana-t 

Current | Using break-all | Using break-word
--- | --- | ---
<img width="375" alt="Screenshot 2021-07-13 at 10 45 01" src="https://user-images.githubusercontent.com/2813677/125430744-864145e8-1636-4585-81d1-b1c2fe1d46a0.png"> | <img width="375" alt="Screenshot 2021-07-13 at 10 45 13" src="https://user-images.githubusercontent.com/2813677/125430722-fe13cc96-a119-41dc-8e33-d80a9ea36f5c.png"> | <img width="375" alt="Screenshot 2021-07-13 at 10 45 23" src="https://user-images.githubusercontent.com/2813677/125430698-056ea527-f63d-4d22-acf4-161ee602a63b.png">

**Comment by boerwastaken** [comment]
Wieso soll die Langwörtafbrechungsprozedurentwicklung schwierig sein den?

**Comment by pedro16v** [comment]
To your point @boerwastaken 

Bad | Better | Works
--- | --- | ---
<img width="374" alt="Screenshot 2021-07-13 at 12 10 00" src="https://user-images.githubusercontent.com/2813677/125442267-a4028d32-094e-4a4b-b1c7-017816108d87.png"> | <img width="375" alt="Screenshot 2021-07-13 at 12 10 26" src="https://user-images.githubusercontent.com/2813677/125442254-ee5fb07d-eb25-4487-8de5-b7818a36b320.png"> | <img width="375" alt="Screenshot 2021-07-13 at 12 11 07" src="https://user-images.githubusercontent.com/2813677/125442245-d1474505-d363-4c4b-9579-095e0e0db4f0.png">

**Comment by boerwastaken** [comment]
is that something that Wagtail does or are you proposing amandments to the German language?
Dutch does this too, btw. Woordaaneenrijging.

(English too — but only sometimes, but I've yet to discover any consistency: battleship, football, chocolate milk, christmas tree).

**Comment by andrewwapling** [comment]
Hi @pedro16v thanks for flagging this one. My preference for readability would be break-word. I'm just wondering whether it would be worth introducing a hyphen on the first line of the broken word to help making the link and align with grammatical conventions, please see reference: https://css-tricks.com/almanac/properties/h/hyphenate/

What do you think about this solution?

**Comment by pedro16v** [comment]
Sounds great @andrewwapling. I'd like to explore more on how hyphenation works for languages other than english. 

From your link, I got into this great article https://alistapart.com/article/the-look-that-says-book/ that mentions the wide use of [Hyphenator](https://github.com/mnater/Hyphenator) (now superseded by [Hyphenopoly ](https://github.com/mnater/Hyphenopoly). It seems to work based on language dictionaries.

**Comment by andrewwapling** [comment]
Hi @pedro16v glad you found the approach interesting. Definitely some fascinating reading to be had. Hyphenation can be a subjective area in English and most likely other languages too. Likewise I'd be interested in learning from other non-language speakers about the suitability of hyphenation. 

I for one am for it from a typographic and typesetting side in English, however cannot personally vouch for it's effectiveness in other language settings. It's main benefit that I can see from a design side is the ability to boost legibility in situations where very long words break lines of text creating undesirable white space, bad forms, orphans and widows. From my perspective in this setting it creates much better readability and legibility (as well as being visually more attractive and cohesive).

I wonder if we were to source some different language examples whether it would help if I mocked up a few course detail screen designs to test it in different contexts? Let me know what you think the best path forward on this one would be?

**Comment by pedro16v** [comment]
I believe that, as a first step, we can ask @megganeturner to move forward with the quick and easy `break-word` change.

In parallel, it would be interesting to study different alternatives as you suggest. Might seem an unimportant detail but readability and comfort play an important role in learning apps, I would argue.

**Comment by andrewwapling** [comment]
@pedro16v sounds like a good approach to me to kick things off. 

I agree whole heartedly with you regarding the importance of ease of readability.

It would be great to run this through some other language subject matter experts to gain insights into how this approach functions in different language groups. We could then test these visually in design prototypes to determine suitability.

**Comment by pedro16v** [comment]
Somehow this fell off the radar and wasn't on the board. @megganeturner please let me know if you can go ahead with this one. For now, it would be just applying the `break-word` rule.
